eng/pipelines/windows.yml

```diff
@@ -117,7 +117,7 @@ jobs:
           - netcoreappWindowsQueues: Windows.7.Amd64.Open+Windows.81.Amd64.Open+Windows.10.Amd64.ClientRS4.ES.Open
```

Comment:
I would  replace `Windows.10.Amd64.ClientRS4.ES.Open` with `Windows.10.Amd64.Client19H1.Open` as PRs should test with latest versions if possible.

Reply:
`ClientRS4.ES` is the only queue that tests on non-English Windows, so I don't think that we want to remove it right now.  I think we want to think more about what Windows versions we test on (e.g. rolling builds might want more of them, since with this change we're testing on 3 of the 5 amd64 versions of Windows 10 we support); and maybe getting a newer non-en-US build should be part of that.

Reply:
> maybe getting a newer non-en-US build should be part of that

+1